.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: build
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
I'm pretty new to workflows but is there a reason you used a wildcard and not `master`?

Reply:
I'm pretty new to workflows too, so this is based from the docs [here](https://github.com/crazy-max/ghaction-docker-meta) :D

As far as I understand it, it's to be future-proof, in case you would use another branch for developing a feature or something, I guess one would want to trigger a build so an image with the latest changes is available for testing. It will also serve as a "build test" I suppose...

I'd agree if you were to prefer only building on master, thus triggering build only when merging from dev into master (for example).

Full syntax doc [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#on)

Reply:
After talking with a friend I've decided to create a `deploy` branch, each time a new version is ready, I'll push there and it will be uploaded to docker hub and GHCR.